engine/execution/computation/computer/computer.go

```diff
@@ -221,9 +207,6 @@ func (e *blockComputer) executeBlock(
 	close(eh.data)
```

Comment:
nit: lets move these after the last commit/hash so that by the time Wait() is called goroutines are already gone.

Reply:
this right now leaks gorountines on error.  i'll fix it in a different pull request.

Reply:
#2661